### DISCUSSION TITLE
Font-size of the ticker for vision impaired people

### DISCUSSION THREAD

**Discussion by shiny-ribbon** [proginosko/LeechBlockNG#486] [Ideas]
Like it says on the title, a font-size option for the ticker would be nice for vision impaired people.
It's as easy as changing the content.css file but of course doing that triggers Chromium's safety mechanisms

**Comment by shiny-ribbon** [comment]
I'm sorry, I made wrongful assumptions. I thought there wasn't any such option in Leechblock, but there is.

**Reply by proginosko** [reply]
No problem. Glad you found it!
